libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushMessaging.java

```diff
@@ -368,36 +388,6 @@ public static void clearRegistrationInfo(Context context, UserAccount account) {
         editor.apply();
     }
 
-    /**
-     * Returns the last backoff time.
-     *
-     * @param context Context.
-     * @return Backoff time.
-     * @param account User account.
-     */
-    @SuppressWarnings("unused")
-    static long getBackoff(Context context, UserAccount account) {
```

Comment:
`backoff` was a package-level member.  I though it was part of the six-days-later push notifications registration retry.  However, that wasn't correct.  Then, I noticed it wasn't used.  Is that correct? I could restore it.